package.json

```diff
@@ -49,8 +49,8 @@
     "lint:scripts": "cd scripts && substrate-dev-run-lint"
   },
   "dependencies": {
-    "@polkadot/api": "^5.8.4-1",
-    "@polkadot/apps-config": "^0.95.1",
+    "@polkadot/api": "^5.9.1",
+    "@polkadot/apps-config": "^0.95.2-114",
```

Comment:
This is going from a stable release (a month ago) to one of the daily beta releases (see https://github.com/paritytech/substrate-api-sidecar/pull/661). I'm assuming that this is OK!